modules/k8s.nix

```diff
@@ -298,7 +298,7 @@ in
     kubeconfig = mkOption {
       description = "path to kubeconfig file (default: use $KUBECONFIG)";
       type = types.nullOr types.str;
-      default = null;
+      default = "$HOME/.kube/config";
```

Comment:
My concern here is whether this would overwrite `$KUBECONFIG`. In some way, maybe that's desirable to be more isolated from the user's environment.